runtime/tests/checker/access_test.go

```diff
@@ -1849,7 +1849,7 @@ func TestCheckAccessImportGlobalValueVariableDeclarationWithSecondValue(t *testi
 		},
 	)
 
-	errs := RequireCheckerErrors(t, err, 5)
+	errs := RequireCheckerErrors(t, err, 7)
```

Comment:
This is a small but unfortunate regression of re-evaluating the middle value of the two-value transfer; the resource capturing error gets reported twice. I think we could just get rid of this by deduplicating the errors list.